# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1298596958 18000
#      Thu Feb 24 20:22:38 2011 -0500
# Node ID 070214996fba4cd0bfea1e50990ce3c30b52da88
# Parent  009068efc66dd3d93835fa136542b4cba49d5e16
subplot.m: Document using INDEX as a vector.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2011-02-24  Ben Abbott  <bpabbott@mac.com>
+
+	* plot/subplot.m: Document using INDEX as a vector.
+
 2011-02-24  Robert T. Short  <rtshort@ieee.org> 
 
 	* polynomial/polyval.m: Properly evaluate constant polynomials.
 
 2011-02-24  John W. Eaton  <jwe@octave.org>
 
 	* strings/strchr.m: Avoid implicit string to number conversion.
 	Bug #32546.  From Kim Hansen <kim@i9.dk>.
diff --git a/scripts/plot/subplot.m b/scripts/plot/subplot.m
--- a/scripts/plot/subplot.m
+++ b/scripts/plot/subplot.m
@@ -47,18 +47,25 @@
 ## +-----+-----+-----+
 ## |  1  |  2  |  3  |
 ## +-----+-----+-----+
 ## |  4  |  5  |  6  |
 ## +-----+-----+-----+
 ## @end group
 ## @end example
 ##
+## @var{index} may be a vector. In which case, the new axis will enclose
+## the grid locations specified. The first demo illustrates an example.
+##
+## @example
+## @code{demo ("subplot", 1)}
+## @end example
+##
 ## @end ifnottex
-## @seealso{plot}
+## @seealso{axes, plot}
 ## @end deftypefn
 
 ## Author: Vinayak Dutt <Dutt.Vinayak@mayo.EDU>
 ## Adapted-By: jwe
 
 function h = subplot (rows, cols, index, varargin)
 
   align_axes = false;
